examples/embedded-nextjs/src/steps/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,40 @@
+import { MonoCheck } from '@kadena/kode-icons/system';
+import { Stack } from '@kadena/kode-ui';
+import { ReactNode } from 'react';
+import * as styles from './Stepper.css';
+
+type StepperProps = {
+  children: React.ReactNode;
+};
+export const Stepper = ({ children }: StepperProps) => {
+  return (
+    <Stack
+      className={styles.stepper}
+      marginInlineStart="sm"
+      flexDirection="column"
+    >
+      {children}
+    </Stack>
+  );
+};
+type StepProps = {
+  status?: 'positive' | 'warning' | 'error' | 'todo';
```

Comment:
nit: I dont' think status is used?

Reply:
that's correct, will only implement them in next pr